.github/workflows/dependabot_pr.yml

```diff
@@ -49,7 +49,7 @@ jobs:
           commit_options: '--signoff'
 
       - name: Update the changelog
-        uses: dangoslen/dependabot-changelog-helper@v1
+        uses: dangoslen/dependabot-changelog-helper@v2
```

Comment:
I think you might need to ask the admin to update the allowed versions under repository settings for this to work.

Reply:
Afaik everything is allowed, but LMK if not and where to change it?

![Screen Shot 2023-01-27 at 3 39 23 PM](https://user-images.githubusercontent.com/542335/215192602-a6cd6129-4742-4b57-8c8a-96355e31ecf8.png)

Reply:
I guess this was a recent change. In the past, we listed out all the actions individually for security reasons.
This is good to know 🙂